docs/build/isc/v1.0.0-rc.6/docs/how-tos/token/register-token.md

```diff
@@ -0,0 +1,59 @@
+---
+description: How to register a native token as ERC20
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - ERC20
+  - EVM
+  - how-to
+---
+
+import Ownership from '../../_admonitions/_ownership.md';
+import Payable from '../../_admonitions/_payable.md';
+
+# Register Tokens
+
+To properly use your native tokens, you should register them as ERC20 using the `registerERC20NativeToken` function from the ISC magic contract.
+
+## Example Code
+
+<Ownership/>
+
+1. First we check if the payed amount to the contract is the same as the [storage deposit](/learn/protocols/stardust/core-concepts/storage-deposit) we want to allow and set the allowance
```

Comment:
```suggestion
1. Check if the amount paid to the contract is the same as the required [storage deposit](/learn/protocols/stardust/core-concepts/storage-deposit)
   and set the allowance.
```

We should probably make this into a reusable phrase like we do with admonitions

Reply:
Wdym? A partially imported markdown?